docs/source/install/index.rst

```diff
@@ -34,10 +42,11 @@ Two additional ESI add-ons provide additional functionality:
    * `python-esileapclient`_
 * `python-esiclient`_: additional OpenStack CLI commands that provide ease of use for users
 
-Instructions on installing these add-ons can be found within the linked repositories.
+Instructions for installing these add-ons can be found within the linked repositories.
 
 .. _install OpenStack: https://docs.openstack.org/install-guide/
 .. _Ironic node multi-tenancy: https://docs.openstack.org/ironic/latest/admin/node-multitenancy.html
+.. _here: https://github.com/CCI-MOC/esi/blob/master/etc/ironic/policy.json.sample
```

Comment:
That file is called "policy.json", but it's not actually a JSON file. Is it misnamed, or is the content incorrect?

Reply:
Is it not? I'm not an expert at json so I don't know if the open/close brackets are needed, but it seems to correspond with this: https://docs.openstack.org/oslo.policy/ussuri/admin/policy-json-file.html

Reply:
Take a look at the [syntax section](https://docs.openstack.org/oslo.policy/ussuri/admin/policy-json-file.html#syntax) of that document...and also note that JSON doesn't support comments.

Reply:
Ah, I see. It looks like when you generate sample policy files in ironic using `tox -egenpolicy`, it can generate json files with the incorrect format; but it'll also generate yaml policy files. I've renamed the sample in this repository to .yaml, and updated the link here. Thanks for catching that!~

Reply:
Looks good to me!